examples/Console/TestOtlpExporter.cs

```diff
@@ -53,12 +54,15 @@ internal static object Run(string endpoint)
 
         private static object RunWithActivitySource(string endpoint)
         {
+            // Enable support for unencrypted HTTP2
+            AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
```

Comment:
This is required when calling insecure (i.e. http not https) gRPC service. This is [no longer required for .NET 5](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-5.0#call-insecure-grpc-services-with-net-core-client).

I can document this in the readme.

Reply:
It would probably be good to have that note in the README.